include/libtorrent/extensions.hpp

```diff
@@ -158,12 +158,11 @@ POSSIBILITY OF SUCH DAMAGE.
 #ifndef TORRENT_DISABLE_EXTENSIONS
 
 #include <vector>
-#include "libtorrent/config.hpp"
-#include "libtorrent/sha1_hash.hpp" // for sha1_hash
-#include "libtorrent/session_handle.hpp"
-#include "libtorrent/peer_connection_handle.hpp"
-#include "libtorrent/span.hpp"
-#include "libtorrent/string_view.hpp"
+
+#include <libtorrent/config.hpp>
+#include <libtorrent/span.hpp>
+#include <libtorrent/string_view.hpp>
+#include <libtorrent/socket.hpp>
```

Comment:
yeah, I've sort of had this old idea of separating the DHT implementation from libtorrent, so I believe libtorrent includes are <> inside the DHT code